src/xdocs/config_imports.xml

```diff
@@ -881,32 +881,56 @@ public class InputIllegalImport { }
       <subsection name="Description">
         <p>Since Checkstyle 4.0</p>
         <p>
-          Controls what can be imported in each package. Useful for
+          Controls what can be imported in each package and compilation unit. Useful for
```

Comment:
please add link to jdk specification to explain what is "compilation unit". To clearly define what we mean and which terminology we use.